WSIST/WSIST.Web/wwwroot/app.css

```
@@ -1046,6 +1046,122 @@ h1:focus {
+    color: var(--text-muted);
+}
+
+.feedback-email {
+    color: var(--text-muted);
+}
+
+.feedback-item-message {
+    font-size: 14px;
+    color: var(--text-primary);
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)